lua/astrocommunity/markdown-and-latex/markdown-preview-nvim/init.lua

```diff
@@ -1,6 +1,15 @@
 return {
   "iamcco/markdown-preview.nvim",
-  build = function() vim.fn["mkdp#util#install"]() end,
+  build = function(plugin)
+    -- HACK: Install using `npx` (using `yarn`) if available
+    ---@see https://github.com/iamcco/markdown-preview.nvim/issues/690#issuecomment-2254280534
+    if vim.fn.executable "npx" then
+      vim.cmd("!cd " .. plugin.dir .. " && cd app && COREPACK_ENABLE_AUTO_PIN=0 npx -y yarn install --frozen-lockfile")
+    else
+      vim.cmd [[Lazy load markdown-preview.nvim]]
+      vim.fn["mkdp#util#install"]()
+    end
```

Comment:
I'm not sure if this should be the default. Any thoughts? @AstroNvim/astrocommunity-maintainers

Reply:
I'm down for this if it means a better out-of-the-box experience for the community, and most of the time I have yarn installed but could this cause issues if people don't have yarn?

If we add this we probs want to include something in the readme

Reply:
This change doesn't require you to have Yarn installed, just npx (NodeJS), and if you doesn't have it, it just go with the old approach.

Reply:
I feel like the default should probably be `yarn` and fall back to `npx`. It also might make sense to contribute this upstream to the plugin itself to improve its installation function. The. We could revert this change and just call their install function

Reply:
> I feel like the default should probably be `yarn` and fall back to `npx`. It also might make sense to contribute this upstream to the plugin itself to improve its installation function. The. We could revert this change and just call their install function

The Markdown Preview repository seems to be unmaintained. I'll make the PR, tho.

Reply:
Ok! We can definitely merge this in the meantime, just like promoting contributions to more projects 🤩

Reply:
> I feel like the default should probably be `yarn` and fall back to `npx`.

I'll make this change to, give me a sec.

Reply:
> lt should probably be `yarn` and fall back to `npx`. It also might make sense to contribute this upstream to the plugin itself to improve its installation function. The. We could revert this change and just call their install function

I agree with this.